src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,23 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <ul>
+    {tabs.map(tab => (
+      <li
+        key={tab.id}
+        className={cn({
+          'is-active': selectedTab.id === tab.id,
+        })}
+        data-cy="Tab"
+      >
+        <a
+          href={`#${tab.id}`}
+          data-cy="TabLink"
+          onClick={() => onTabSelected(tab)}
```

Comment:
The onClick function may cause a page refresh because it doesn't prevent the default action for the anchor tag. Use event.preventDefault() to prevent this.